settings/version

```diff
@@ -1 +1 @@
-2.0rc8
+2.0 %GIT_COMMIT%
```

Comment:
is this now the final version 2.0? Otherwise the `rc8` should in my opinion be kept in the version string.

Reply:
As this goes into develop and 2.0 will be the next release we could just go for 2.0 except we expect another RC?